data/json/items/comestibles/meat_dishes.json

```diff
@@ -1830,7 +1834,7 @@
     "subtypes": [ "COMESTIBLE" ],
     "id": "paella_valenciana",
     "name": { "str_sp": "Valencian paella" },
-    "conditional_names": [ { "type": "COMPONENT_ID_SUBSTRING", "condition": "mutant", "name": { "str_sp": "\"deluxe\" beans and rice" } } ],
+    "conditional_names": [ { "type": "COMPONENT_ID_SUBSTRING", "condition": "mutant", "name": { "str_sp": "vulgar paella" } } ],
```

Comment:
maybe mention in the PR that you also changed this since it's not related to soup, for bookeeping